crates/database/src/orders.rs

```diff
@@ -569,6 +569,31 @@ pub fn limit_orders_with_outdated_fees(
         .fetch(ex)
 }
 
+/// Count the number of valid limit orders belonging to a particular user.
+pub async fn count_limit_orders(
+    ex: &mut PgConnection,
+    min_valid_to: i64,
+    owner: &Address,
+) -> Result<i64, sqlx::Error> {
+    const QUERY: &str = const_format::concatcp!(
+        "SELECT COUNT(*) FROM (",
+        "SELECT ",
+        ORDERS_SELECT,
```

Comment:
If you only want to return the count anyway the `ORDERS_SELECT` seems like unnecessary overhead.

Reply:
Oh, yeah. Didn't look closely enough. 🤦‍♂️